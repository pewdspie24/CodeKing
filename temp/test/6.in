17
33
874430152 59727516 622182909 91170285 135798953 321316376 708979111 566438194 510716316 699147259 99061593 77333216 838454696 597222414 918623339 161984408 731413504 996599289 302981766 50155085 242236937 409132742 538634471 128538916 515043572 207585618 692195583 576676566 383086541 813220757 307514947 322481603 892943940 
23
218785082 733174964 822210553 1595098 954091060 755507309 845475650 286430767 507638686 950097919 172000424 959435764 763904375 432209537 311262480 165773257 941421678 56836947 160936546 95886127 88775019 181894238 899299972 
43
297842472 751071949 808400288 50036539 335683808 938153508 157668475 682342435 360418998 242241512 674186269 918350322 31162511 324618164 968451130 94434821 214263705 51791884 622759699 720446040 630987187 622678037 127202515 115889780 42483795 251152773 120007120 704519141 586879665 941933152 266497540 564189009 779988815 173028155 328978898 518316871 698152581 48551540 289885667 328664359 78848052 773611277 394632848 
24
680712923 923079062 949131210 269409529 468788274 8689223 670411270 371767955 476846720 808413952 274945641 418780998 803275915 479093941 102826123 760694550 172761158 291330412 232468076 404568562 781031007 157532372 291614471 434657274 
22
201297827 374119957 299987054 637325255 636858535 667680393 272900139 210188206 309814222 161035637 155434094 616429755 927604336 929569685 562710682 577452528 686510210 24974313 329920370 230002210 971424592 162821786 
24
436897298 591250257 275312809 36592474 590855370 377947507 226734383 43146738 235107753 840626273 756734759 985034265 888100279 735350048 657910682 764099081 802131585 485065097 737355069 894009027 474474857 192570301 9761576 33846764 
49
248728799 208847039 880567934 369937927 400866736 518360452 181785128 359688896 323253879 332917359 834163899 118750263 713491911 133302265 436291742 200339902 849288118 348593898 729998017 986132401 92609127 683053748 679384668 440619009 324502757 633860278 214252552 618458135 410135447 480441577 365599458 542943126 358348301 242111720 283306732 876239839 121875828 979290160 460365557 532259086 332583355 923514720 315572533 142129182 471306931 806957530 890664382 949072576 595369224 
40
361606926 761125514 80878593 197896377 102465866 255281291 104083083 195963383 143678883 880992200 33648538 234674145 222559324 386988801 728784908 68003378 244963556 147828235 2328836 138837337 103761619 147504438 510860797 903074463 92531394 138364428 729662421 875584381 27806703 933389961 362418234 697047853 653914021 414757201 253297133 136992546 212524237 290698121 34820312 608487873 
45
967578770 528329121 450255254 914783333 968796025 972791676 984714097 388694754 732806519 899316028 138540518 758205817 427046746 179488822 793158410 137638978 283019521 911766203 451725595 46892358 114032557 64439324 33805781 304290655 164375397 492143520 440899420 872432576 426736919 180443325 486163119 791943032 932573003 528264964 514499079 341061158 345531980 614519736 729178888 169508214 155548210 399374264 825609586 717324284 692115721 
26
24324937 429117619 610046920 383494835 408411908 482704935 477762676 524471973 186362864 555156296 328652001 189037376 123261662 547774568 48585662 579141290 282889724 272586279 740346494 410885908 260298072 584872117 23322987 367595028 418953325 574493115 
30
822573768 45658292 456662831 81619310 326559644 727725238 747406875 480626685 509996111 212392508 739454334 240308172 217864913 966504051 580140384 206455187 451410478 83335181 808504679 618890949 104061763 408539824 306482287 35110212 754179362 40690280 531226421 80540134 756103729 914102361 
43
579099507 825592616 278149863 533914457 427375756 132508537 476258996 318648005 160670898 94978576 173410443 728021105 748999691 328588710 745438756 518520936 94538341 91825243 119440683 60335890 902248899 810842015 404002513 548293606 685514818 369728739 311695366 431347339 616995913 115107649 582753028 327530979 289091533 167782257 606824110 59355004 708859023 518028743 118059381 65385221 652814538 883044719 936283932 
31
168417702 348466699 314033549 225756310 356551449 954331188 984450664 721158361 366672531 434798880 974885826 882423280 182690261 852192064 362091342 986850087 554653091 703084156 206753003 414436459 884886702 705434516 271232583 243049296 751670701 511099121 795477247 380802189 702374798 338251225 753934371 
27
52348510 714849769 872198115 834074946 908329900 835757562 357111973 831855388 380605524 76370676 654496041 664750237 710255704 737753046 271823982 45822774 575958048 31858748 485027068 818829541 752941681 980350955 797213146 273285051 110747998 49580047 503126299 
39
907580824 376350913 295710276 193611076 22438248 490157867 278709433 750716117 610381112 197444119 38337130 806255703 122730952 259029290 905551558 286917897 279641897 602294870 562434447 478291915 301667297 563000269 317690046 630328599 618579208 175250714 239758198 186392160 667731567 927170691 492402039 111491053 330662670 94276009 41645723 530348691 722288357 279951913 875308554 
22
690359996 653526494 946888593 570590268 414607938 606290404 93694784 291771153 346930598 468183485 699501324 820908816 122253995 502700273 685001709 907120040 499905936 965010758 682363452 321396306 40797007 320939192 
45
423663042 946591437 227577055 9390075 765241353 610453842 413235829 146154562 570675100 149617490 134082825 572148619 63291380 499535906 232000203 259890065 195185679 956162947 507426239 547682958 336399030 731349403 12043510 989133508 806950565 299174443 415384038 442695131 736085836 311693517 988478078 709856834 801006348 394511317 289806206 82868834 271692660 680684895 241972734 419273664 915860983 907184097 722214887 208731280 171181184 
