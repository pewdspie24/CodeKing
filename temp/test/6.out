Bi
Lema
Lema
Bi
Lema
Bi
Bi
Bi
Bi
Lema
Lema
Lema
Lema
Lema
Lema
Lema
Bi
